www/js/configs/app.config.js

```diff
@@ -98,6 +98,14 @@ angular.module("binary").constant("config", {
             category: "OVER/UNDER"
         }
     ],
+    supportedTradeTypes: [
```

Comment:
There are 7 trade types in TickTrade

{
    Volatility: [
        "Digits Digit Matches/Digit Differs",
        "Digits Digit Odd/Digit Even",
        "Digits Digit Over/Digit Under",
        "Higher/Lower",
        "Rise/Fall",
        "Asians"
    ],
    Forex: ["Up/Down"]
}